src/inspector/view/qml/MuseScore/Inspector/text/TextInspectorView.qml

```diff
@@ -322,31 +322,23 @@ InspectorSectionView {
             }
         }
 
-        PopupViewButton {
-            id: textAdvancedSettingsButton
+        ExpandableBlank {
+            id: showItem
+            isExpanded: false
 
-            width: contentColumn.width
-            anchorItem: root.anchorItem
+            title: isExpanded ? qsTrc("inpsector", "Show less") : qsTrc("inspector", "Show more")
+
+            visible: root.model ? !root.model.isEmpty : false
+            width: parent.width
 
             navigation.panel: root.navigationPanel
             navigation.name: "TextAdvancedSettings"
             navigation.row: insertSpecCharactersButton.navigation.row + 1
 
-            text: qsTrc("inspector", "More…")
-            visible: root.model ? !root.model.isEmpty : false
-
-            popupContent: TextSettings {
+            contentItemComponent: TextSettings {
                 model: root.model
-
-                navigationPanel: textAdvancedSettingsButton.popupNavigationPanel
-            }
-
-            onEnsureContentVisibleRequested: function(invisibleContentHeight) {
-                root.ensureContentVisibleRequested(invisibleContentHeight)
-            }
-
-            onPopupOpened: {
-                root.popupOpened(popup, control)
+                navigationPanel: root.navigationPanel
+                navigationRowStart: showItem.navigation.row
```

Comment:
To be on the safe side, this should be <code>showItem.navigation.row <b><i>+ 1</i></b></code>. I know that `TextSettings` has its first control at `navigationRowStart + 1`, so there is no real problem yet, but in general, `navigationRowStart` is supposed to be the first index that is safe to be used inside the section, so that should be `showItem.navigation.row + 1`.